README.md

```diff
@@ -95,53 +98,80 @@ $5 ==> "Vernie Schmidt"
 
 ```java
 Faker faker = new Faker();
-faker.expression("#{letterify 'test????test'}"); // testqwastest
-faker.expression("#{numerify '#test#'}"); // 3test5
-faker.expression("#{templatify 'test','t','q','@'}"); // @esq
-faker.expression("#{examplify 'test'}"); // ghjk
-faker.expression("#{regexify '[a-z]{4,10}'}"); // wbevoa
-faker.expression("#{options.option '23','2','5','$','%','*'}"); // *
-faker.expression("#{date.birthday 'yy DDD hh:mm:ss'}"); // 61 327 08:11:45
-faker.expression("#{csv '1','name_column','#{Name.first_name}','last_name_column','#{Name.last_name}'}");
+faker.
+
+expression("#{letterify 'test????test'}"); // testqwastest
+faker.
+
+expression("#{numerify '#test#'}"); // 3test5
+faker.
+
+expression("#{templatify 'test','t','q','@'}"); // @esq
+faker.
+
+expression("#{examplify 'test'}"); // ghjk
+faker.
+
+expression("#{regexify '[a-z]{4,10}'}"); // wbevoa
+faker.
+
+expression("#{options.option '23','2','5','$','%','*'}"); // *
+faker.
+
+expression("#{date.birthday 'yy DDD hh:mm:ss'}"); // 61 327 08:11:45
+faker.
+
+expression("#{csv '1','name_column','#{Name.first_name}','last_name_column','#{Name.last_name}'}");
```

Comment:
Hmm...
I'm pretty sure all these and most of other changes have nothing to do with the actual PR description. Please stick only to those changes which are solving the issue mentioned in PR description. If some other things should be changed we can discussed it in separate PRs